### ISSUE TITLE
Lessons not showing, even with postname permalink

### ISSUE THREAD

**Issue by tganey** [LearnPress/learnpress#411]
Lesson content does not show upon page loading. I just get a blank page. I've tried several remedies found on different forums (making sure permalink settings in WP are postname, settings-courses link settings in Learnpress, turned off all other plugins, cleared database in LP tools, changed theme to WP twenty twenty, etc.). I've run out of ideas. Learnpress 3.6.2.5 with WP Twenty Seventeen theme. I'm testing using all the sample course data. Can you help resolve? www.tampayogatherapy.com/courses/

**Comment by J4ir0** [comment]
H! I have same issue, how can you solve?

**Comment by tganey** [comment]
My particular issue (described above) completely resolved when I changed the theme. Why on earth Learnpress doesn't work with a basic WP themes like Twenty Seventeen, I cannot imagine. But once I changed from WP Twenty Seventeen to a different theme, all became right with the universe! I read about this solution in a different forum ... it never occurred to me that a WP standard theme would not be supported by the developers of LP. But there you go. Hope it helps.
